aws-lc-rs/src/cipher.rs

```diff
@@ -347,10 +355,14 @@ define_cipher_context!(DecryptionContext, EncryptionContext);
 /// Cipher algorithm identifier.
 pub enum AlgorithmId {
     /// AES 128-bit
-    Aes128,
+    Aes128 = 0,
+
+    // Discriminant explicitly set due to changing implicit discriminant would break semver
+    /// AES 192-bit
+    Aes192 = 2,
 
     /// AES 256-bit
-    Aes256,
+    Aes256 = 1,
 }
```

Comment:
I feel like this add more overhead then just living with the fact that they aren't sorted in-order by AES key size? It's an aesthetic thing that arguably a documentation system could handle.